db/models.py

```diff
@@ -14,3 +14,34 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
```

Comment:
you still have wrong `related_name`

Reply:
Hi! What names threre should be? I tested it and got queryset with actors and genres respectively.

Reply:
fix